README.md

```diff
@@ -144,7 +144,7 @@ GitHub Actions will verify that any code changes are done in a style compliant
 way. Install `clang-format` and `mint`:
 
 ```
-brew install clang-format
+brew install clang-format@11
```

Comment:
The `@11` appears to be unnecessary here; I `brew upgrade clang-format` on an existing installation brought me to 11.0.0.

Reply:
I added it to the readme to be consistent with our scripts. Ideally we would like to lock on a particular version of clang-format to avoid unexpected CI breakages like today. specifying `@11` is the best we can do now with the way clang format team handles their releases in Homebrew.